app/components/trainees/outcome_details/view.html.erb

```diff
@@ -1,6 +1,6 @@
 <%= render SummaryCard::View.new(
   trainee: data_model.trainee,
-  title: "QTS details",
+  title: "#{data_model.trainee.training_route_manager&.award_type} details",
```

Comment:
Since we're always 'looking through' the trainee's `training_route_manager` to find the award type, we can delegate the `award_type` method to the `training_route_manager` in the trainee model.

Reply:
I see you've used a `&` here - what happens when `data_model.trainee.training_route_manager` is nil? We should be handling that somehow.